scripts/load-test-workspaces/README.md

```diff
@@ -26,8 +26,8 @@ const API_TOKEN = process.env.API_TOKEN;
 const API_HOSTNAME = ''; // 'XXXXXXXXXX.execute-api.<region>.amazonaws.com'
 const REGION_NAME = '';
 const REGION_SHORT_NAME = ''; // find the exhaustive list at ../../main/config/settings/.defaults.yml
-const STAGE_NAME = ''; // 'sw-dev'
-const SOLUTION_NAME = 'sw';
+const STAGE_NAME = ''; // 'changeme'
+const SOLUTION_NAME = 'changeme';
```

Comment:
Is it really necessary to change? Wonder if customers had already copied strings 'sw' and 'sw-dev' and are using them as defaults already.

Reply:
No you're right. Reverted.